docs/docs/reference/settings.md

```diff
@@ -405,6 +404,21 @@ root:
   handlers: [console, file]
 ```
 
+### <a name="cli-cwd"></a>`cli.cwd`
+
+- `meltano` CLI option: `--cwd`
+- Default: `Current directory`
+
+With `--cwd` option you can configure meltano as if had been started in a specified directory.
+
+#### How to use
+
+Specify path to a directory which contains a valid [ `meltano.yml` project file](/concepts/project#meltano-yml-project-file).
+
+```bash
+meltano --cwd = /path/containing/meltano_yml
```

Comment:
Using `=` with spaces like this doesn't work, and the value should be quoted so that if a user tries to do this with a path that e.g. has spaces in it it'll still work.

```suggestion
meltano --cwd '/path/containing/meltano_yml'
```